### DISCUSSION TITLE
urp 렌더러 적용

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#1063] [5. Q&A]
안녕하세요 유니티에서 URP 렌더러를 적용하고 월드 테스트 하면 마젠타로 나오는데
제페토에서 urp 렌더러는 사용 불가능 하나요?? ㅠㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 아쉽게도 URP는 아직 지원하지 않고 있습니다. 
사용 가능한 패키지는 아래 가이드를 참고 부탁 드립니다. 
https://studio.zepeto.me/guides/unity_package_list

감사합니다.

**Reply by yj098** [reply]
ㅠoㅠ 그렇군요 답변 감사합니다 :)
